server/src/main/java/org/candlepin/resource/StatusResource.java

```diff
@@ -117,6 +130,8 @@ public Status status() {
 
         Status status = new Status(good, version, release, standalone,
             jsProvider.getRulesVersion(), jsProvider.getRulesSource());
+
+        statusCache.put("status", status);
```

Comment:
@barnabycourt This is what we talked about a while ago, caching of /status for 5 seconds. Just wanted to point you to this, in case you have any concerns. (this PR will have much less code after all other caching PRs get merged)